app/views/dashboard/_sidebar.html.erb

```diff
@@ -1,5 +1,57 @@
-<br />
+  <br />
+ 	<div class="panel panel-default">
```

Comment:
Hi - this is looking real good -- but do you think you could make the indentation really consistent here? It makes it a lot easier to read and helps keep our code clean in the future. Thank you!!!